src/helpers.ts

```diff
@@ -37,24 +42,36 @@ function checkXpub(xpub: string) {
   const prefix = xpub.substring(0, 4);
 
   if (prefix === 'xpub') {
-    network.type = BITCOIN_NETWORK;
+    configuration.network = BITCOIN_NETWORK;
   }
   else if (prefix === 'Ltub') {
-    network.type = LITECOIN_NETWORK;
+    configuration.network = LITECOIN_NETWORK;
   }
```

Comment:
As you add more coins, you might want to aim for something like

```javascript
const xpubPrefix = xpub.substring(0, 4);
const network = NETWORKS.find( ({ prefix }) => prefix === xpubPrefix );
if (!network) { throw ... }
```

Reply:
Yes, I agree that this aspect will have to be refactored in the future.